### ISSUE TITLE
Unit test fails after arcade update

### ISSUE THREAD

**Issue by AdamYoblick** [dotnet/winforms#1030]
https://dev.azure.com/dnceng/public/_build/results?buildId=198756&view=ms.vss-test-web.build-test-results-tab&runId=4820080&resultId=101525&paneView=debug

This is passing in master.

No idea why an arcade update would cause this to fail, but I'm disabling this test for now to allow code to flow.

**Comment by hughbe** [comment]
The build is not found. I suggest closing this :))

**Comment by weltkante** [comment]
> but I'm disabling this test for now to allow code to flow.

Does it require to reenable a test for closing this issue? Probably should document which test needs to be reenabled?

**Comment by RussKie** [comment]
> > but I'm disabling this test for now to allow code to flow.
> 
> Does it require to reenable a test for closing this issue? Probably should document which test needs to be reenabled?

Yes, once disabled test(s) is re-enabled an associated issue(s) get closed.
I think when we started, flaky tests would get commented out. Later we started using "Skip(reason, link)" notation.

Looks like @hughbe has found the offending test.
